core/src/main/java/io/camunda/migrator/RuntimeMigrator.java

```diff
@@ -68,7 +69,8 @@ public class RuntimeMigrator {
 
   protected boolean retryMode = false;
 
-  protected int batchSize = MAX_BATCH_SIZE;
+  @Value( "${migrator.batch-size:500}" )
```

Comment:
```suggestion
  @Value("${migrator.batch-size:" + DEFAULT_BATCH_SIZE + "}")
```
You need to rebase against `main` to get the latest changes.